README.md

```diff
@@ -1 +1,21 @@
-# hello-world
\ No newline at end of file
+# "Hello Microverse" project
+
+> This is the first project on Microverse. The main tasks are setting up a "Hello world" repository and creating .html and .css files.
+
+## Built With
+
+- HTML & CSS
+
+## Authors
+
+👤 **Khusniddin Ismoilov**
+
+- GitHub: [@KIsmoilov](https://github.com/KIsmoilov)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
```

Comment:
Very nice to add the readme file. However it is missing a few headers. 
#### 1. Getting started 
This is very important since it helps to educate other people on how to set up and run the project
#### 2. Acknowledgements
This helps to acknowledge all contributions and contributers including those whoose code or materials were used indirectly
#### 3. License
This will help decide what people are allowed to do with your project, add value and makes it look more proffessional.

_Kindly make sure that all headers from the [template](https://github.com/microverseinc/readme-template) should be present. Always remember that the quality of a README file often differentiates a good project from a bad project._